base/lttextcomp.dtx

```diff
@@ -1527,14 +1527,11 @@
 %
 %    \begin{macrocode}
 %<*TS1sty>
-\providecommand\DeclareRelease[3]{}
-\providecommand\DeclareCurrentRelease[2]{}
-
-\DeclareRelease{}{2018-08-11}{textcomp-2018-08-11.sty}
```

Comment:
I would keep that line (even though technically not necessary) and also add a line of docu like "For any rollback request before 2018-08-11 we make an attempt by loading the 2018  version."

Reply:
Added back that line.